testsuite/pytests/test_aeif_cond_alpha_multisynapse.py

```diff
@@ -37,20 +28,31 @@
 
 @pytest.mark.skipif_missing_gsl
 class TestAeifCondAlphaMultisynapse:
+    r"""
```

Comment:
no reason to make this a raw string.
```suggestion
    """
```

Reply:
It's afaik a good convention to make all docstrings triple-quoted and raw, because if LaTeX math syntax is used inside the docstring, it could contain strings that would otherwise be parsed as escape characters (for example, "\nu").

Reply:
@jessica-mitchell, if this is required, we should add something in the code style docs and ideally to the docstyle linter. Could you keep an eye open how such linting is done in other projects?

Reply:
I've opened a new Issue #3262 for this, so I would be OK to merge this now and handle the syntax consistently with other models in a separate PR.